internal/state/mailbox.go

```diff
@@ -301,7 +301,7 @@ func (m *Mailbox) Copy(ctx context.Context, seq *proto.SequenceSet, name string)
 // Move moves the messages represented by the given sequence set into the mailbox with the given name.
 // If the context is a UID context, the sequence set refers to message UIDs.
 // If no items are moved the response object will be nil.
-func (m *Mailbox) Move(ctx context.Context, seq *proto.SequenceSet, name string) (response.Item, error) {
+func (m *Mailbox) Move(ctx context.Context, seq []command.SeqRange, name string) (response.Item, error) {
```

Comment:
Wait why does Move take a SeqRange and not a SeqSet? 🤔

Reply:
`SeqSet == []SeqRange`

Reply:
Ah okay, must have misread, i thought when i left that comment that Copy accepted SeqSet and Move accepted SeqRange, and wondered why they didn't accept the same type. My bad.